lib/fluent/plugin/filter_process_snmptrap.rb

```diff
@@ -38,31 +39,44 @@ def start
     end
 
     def filter(tag, time, record)
+      @time = time
+      @tag = tag
       message = record.to_s
       message = message.delete('\\"')
       snmp_msg = message.gsub(/(?:(SNMPv2-(\w+)(::)(\w+)((\.)(\d+)){1,13}(=>))|(host=>))/, "")
       record["machineId"] = ""
-      record["rmc_host"] = ""
+      record["rmcHostIP"] = ""
       record["event"] = ""
       record["status"] = ""
-      record["sensor"] = ""
+      record["device"] = ""
       record["severity"] = ""
       record["sensorValue"] = ""
       record["error"] = ""
       record["message"] = ""
-      record["timestamp"] = ""
+      record["time"] = ""
```

Comment:
I don't think we need time, fluentd should already have its timestamps.
Even if it doesn't, Geneva and service bus will